samples/consul-ui/consul_services.json.erb

```diff
@@ -6,6 +6,19 @@
 #   INSTANCE_EXCLUDE_TAG: Exclude instances having the given tag (default: canary)
 #   EXCLUDE_SERVICES: comma-separated services regexps to exclude (default: lbl7.*,netsvc-probe.*,consul-probed.*)
 
+  unless @sort_consul_service_nodes
```

Comment:
I think this could be moved to an helper class, should we add other preferences one day, to avoid duplication of this rather long sequence to load them